src/main/java/Group15/MuscleSelectionView.java

```diff
@@ -0,0 +1,124 @@
+package Group15;
+
+import Group15.Api.BodyPart;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.*;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class MuscleSelectionView {
+
+    public static Scene createMuscleSelectorScene() {
+        VBox vBox = new VBox();
+        HBox inputAndEquipBox = new HBox();
+        HBox imageAndTextButtons;
+
+        ComboBox equipmentSelector = new ComboBox<>();
+        equipmentSelector.getItems().addAll(
+                "All",
+                "Body weight",
+                "Barbell",
+                "Dumbbell",
+                "Machine"
+        );
```

Comment:
Should allow for multiple choices, ofc not combined with all

Reply:
I am once again asking for assistance, on this one